marketing_crm_partner/models/crm_lead.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Tecnativa S.L. - Jairo Llopis
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2016 Tecnativa S.L. - David Vidal
+# Copyright 2018 Tecnativa S.L. - Cristina Martin R.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class CRMLead(models.Model):
+    _inherit = "crm.lead"
+
+    @api.model
+    def _prepare_customer_values(self, name, is_company, parent_id=False):
+        """Populate marketing fields in partner."""
+        res = super()._prepare_customer_values(name, is_company, parent_id)
+        # We use self.env['utm.mixin'] for not losing possible overrides
+        # see https://github.com/odoo/odoo/blob/
+        # e5c8071484c883bf78478a39ef2120bcd8f2442d/addons/utm/models/utm.py#L51
```

Comment:
Update the link to match the current version code: https://github.com/odoo/odoo/blob/51833e4735fb0b761d3cd2867dfd166813469e70/addons/utm/models/utm_mixin.py#L52